API/src/utils/Password/index.js

```diff
@@ -0,0 +1,32 @@
+const { resolve } = require('app-root-path');
+const bcrypt = require ('bcrypt');
+
+// const comparePasswords = async function (password, storedPassword, callBack ) {
+  //const comparePasswords = async function (password, storedPassword) {
+    // let isSamePassword = await bcrypt.compare(password, storedPassword, function(err, result) {
+    //   console.log('result');
+    //   console.log(result);
+    //   resolve(''+result);
+    //   return result;
```

Comment:
I was not able to make this return statement return the actual Boolean value, that is why I am using a call back function.